Sources/SwiftSDKCommand/ListSwiftSDKs.swift

```diff
@@ -50,8 +50,15 @@ package struct ListSwiftSDKs: SwiftSDKSubcommand {
             return
         }
 
-        for artifactID in validBundles.sortedArtifactIDs {
-            print(artifactID)
+        let sdkBundles = validBundles.listSwiftSDKs(hostTriple: hostTriple)
+        for bundle in validBundles.sortedArtifactIDs {
+            if let sdkTriples = sdkBundles[bundle], !sdkTriples.isEmpty {
+                print("SDK bundle: \(bundle)")
+                print("SDKs:")
```

Comment:
Ditto, this should be hidden behind a flag to preserve compatibility with existing scripts.

Reply:
Same as above, this is such important info that users want, that nobody will complain that adding more info breaks a few scripts.